db/queries/statuses/queries.py

```diff
@@ -243,3 +245,10 @@ def update_statuses(application_id: str, form_name: str, is_summary_page_submitt
 
     update_application_status(application, round.feedback_survey_config)
     db.session.commit()
+    if isinstance(round.deadline, datetime):
+        deadline = datetime.strptime(round.deadline.strftime("%Y-%m-%dT%H:%M:%S"), "%Y-%m-%dT%H:%M:%S")
+    else:
```

Comment:
Why do we need this `if`statement? We have written the time to the database, what format do we use to write it?

Reply:
Fixed